### DISCUSSION TITLE
How to divide a single block into two (split the single block into two)

### DISCUSSION THREAD

**Discussion by hsuraweera** [JiHong88/SunEditor#1160] [General]
How to divide a single block into two (split the single block into two) when we use the editor in GUI mode?

![image](https://user-images.githubusercontent.com/90305373/216953358-b9ca11ea-02ac-43e2-9dac-ea2309aaca0f.png)

**Comment by JiHong88** [comment]
Do you mean about like this?
https://ckeditor.com/docs/ckeditor5/latest/examples/framework/multi-root-editor.html

**Reply by hsuraweera** [reply]
Yes, I can divide vertically (It is showing when the 'show block' button is clicked) but didn't able to do it horizontally.
![SunEditor](https://user-images.githubusercontent.com/90305373/217422334-f11d3858-a05b-41b5-8928-0406deda07fc.png)

**Reply by hsuraweera** [reply]
@JiHong88 any update please?

**Reply by JiHong88** [reply]
@hsuraweera Sorry, This is difficult to implement due to the structure of the editor.
